package.json

```diff
@@ -52,7 +52,7 @@
     "spatial"
   ],
   "license": "Apache-2.0",
-  "main": "dist/index.js",
+  "main": "src/index.js",
```

Comment:
That's a mistake, will fix.

Reply:
But really why do we need to build this library?

Reply:
But really, why do we need to build it?

Reply:
Fair question;  I think the reason it was started was so that it was backward compatible with older versions of Node, that didn't support things like `const`, `let` etc.  We could remove it, but might want to wait until a major version bump.